core/capabilities.go

```diff
@@ -78,8 +78,8 @@ const (
 	Capability_ImageToVideo
 	Capability_Upscale
 	Capability_AudioToText
-	Capability_LLM
 	Capability_SegmentAnything2
+	Capability_LLM
```

Comment:
Personally, I'd be more explicit and give each value an explicit int value. Likt it's done in protobufs. For example:
```
Capability_H264 = 0
Capability_MPEGTS = 1
....
```

Reply:
Discussed in Discord. I've pushed a commit with this change.